azure-devops/azext_devops/dev/repos/arguments.py

```diff
@@ -57,3 +57,7 @@ def load_code_arguments(self, _):
     with self.argument_context('repos import create') as context:
         context.argument('repository', options_list=('--repository', '-r'))
         context.argument('git_source_url', options_list=('--git-source-url', '--git-url'))
+
+    with self.argument_context('repos ref') as context:
+        context.argument('repository', options_list=('--repository', '-r'))
```

Comment:
nit: Defining 'repository' alias is not needed now. We have added it to the parent command group so it applies to all commands now.

Reply:
It's the `repos pr` group that has the `repository` alias, not the `repos` one.